doc/manual/configuration.rst

```diff
@@ -1195,3 +1209,37 @@ Specifies alias names for packages::
 
    alias:
       myApp: "host/files/group/app42"
+
+build / develop mode arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set default build arguments here. See :ref:`manpage-dev` or
+:ref:`manpage-build` for details.::
+
+
+    dev:
+        no-logfile: True
+        build-only: True
+    build:
```

Comment:
Isn't it

```
config:
   dev:
      ...
```
? I'm thinking if `command` would be more appropriate. `config` is quite generic.